extensions/picocli/runtime/src/main/java/io/quarkus/picocli/runtime/PicocliCommandLineProducer.java

```diff
@@ -0,0 +1,63 @@
+package io.quarkus.picocli.runtime;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Instance;
+import javax.enterprise.inject.Produces;
+import javax.enterprise.inject.literal.NamedLiteral;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+import io.quarkus.arc.DefaultBean;
+import io.quarkus.picocli.runtime.annotations.TopCommand;
+import picocli.CommandLine;
+
+@ApplicationScoped
+public class PicocliCommandLineProducer {
+
+    private final Instance<Object> topCommand;
+    private final PicocliConfiguration picocliConfiguration;
+
+    @Inject
```

Comment:
This is redundant since Arc knows to use the constructor even without the annotations